context.tests/ContextTestFiles/ContextSimple1.AdditionalElements

```diff
@@ -0,0 +1,39 @@
+From: Joe <Joe@hobo.lab>
+Date: Sat, 08 Jul 2017 11:30:20 -0700
+Subject: Need more memory
+Message-Id: <34C73MNYE0U4.6M64WULRWE982@SSORDSK-JSHO02>
+To: Jean <Jean@opsstation.lab>
+X-Direct-Context: <2ff6eaec83894520bbb872e5671ff49e@hobo.lab>
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="=-1GehjmVTXFcPRv+tjzABIA=="
+
+--=-1GehjmVTXFcPRv+tjzABIA==
+Content-Type: text/plain; charset=utf-8
+
+Simple Body
+--=-1GehjmVTXFcPRv+tjzABIA==
+Content-Type: text/plain
+Content-Disposition: attachment; filename=metadata.txt
+Content-Transfer-Encoding: quoted-printable
+Content-ID: <2ff6eaec83894520bbb872e5671ff49e@hobo.lab>
+
+version: 1.1
+id: <2142848@direct.example.com>
+patient-id: 2.16.840.1.113883.19.999999:123456
+type: radiology/report
+purpose: research
+creation-time: 20210416080510.1245-4000
+patient: givenName=3DJohn; surname=3DDoe; middleName=3DJacob; dateOfBi=
+rth=3D1961-12-31; gender=3DM; postalCode=3D12345
+ihe-format-code=ihc: pcc/HL7+N0D/ADT_A01/2.5.1
+content-type: 79429-7/Admission notification note/urn:oid:2.16.840.1.1=
+13883.6.1/LOINC
```

Comment:
The two lines noted are the considered the same line.  To enabled this the second line would start with a space.  I know we are just adding this extra metadata to test this does not break the 1.1 parser.  But we still need that space to keep the semantic meaning as described in section 3.0 Metadata: Content.

Reply:
Sure, I can update.   I've not seen this before, and was using line 26 and 27 as an example, and I didn't see a space there so I didn't look into the format

Reply:
Mimekit actually doesn't support the space as the first character to indicate a new line.